mongorm/fields/DateTimeField.py

```diff
@@ -37,5 +37,8 @@ def fromPython( self, pythonValue, dereferences=[], modifier=None ):
 	
 	def toPython( self, bsonValue ):
 		if PYTZ and bsonValue is not None:
-			bsonValue = utc.localize( bsonValue )
+			if bsonValue.tzinfo is None:
+				bsonValue = utc.localize( bsonValue )
+			elif bsonValue.tzinfo != utc:
+				bsonValue = bsonValue.astimezone(utc)
```

Comment:
This looks like it makes the function consistently return non timezone aware datetimes in UTC timezone. But why is the input inconsistent?

Reply:
because if `bsonValue` has `tzinfo`,  `utc.localize` will raise `ValueError: Not naive datetime (tzinfo is already set)`.

Reply:
I guess this is because in some place we use pymongo to set the datetime field value, but for my case, I need to initialise the resource through the mongorm, like this `EnrolmentResource(model=Enrolment()._fromMongo(enrolment_doc))`. I dont know if there is an other way to fix that issue.

Reply:
Right OK.

An alternative is to configure pymongo to use non timezone aware datetimes as in `EnrolmentController.mongorm_models_from_query`, but this is probably a better solution.